### PR TITLE
Minorfixes

### PR DESCRIPTION
**Pull request by togfoxy**
Changes proposed in this pull request.
-
-
-

### CODE REVIEW

**Comment by togfoxy** [comment]
Fixes #223

**Comment by togfoxy** [comment]
So, as usual, I wanted to change one thing and came across about six small things.

This PR comes after Phil's reformatting PR.

**Comment by togfoxy** [comment]
If Phil's formatting PR is merged, then this can go next.

Feel free to review and do an immediate merge if it looks right. There are a lot of tiny changes in this one but nothing major. Thanks.

File: source/objects/base.lua [hunk 1]
```diff
@@ -13,7 +13,7 @@ local Base = {}
 -- Local Variables
 -- ~~~~~~~~~~~~~~~~
 
-local landingLights     = Assets.getImageSet("landingLights")
+local landingLights     = Assets.getImageSet("landingLights2")
 landingLights.animation = Assets.newAnimation("landingLights", landingLights.image, 64, 8, 1, '1-4', 0.5)
```
Comment (MadByteDE) [inline source/objects/base.lua:17]: Why renaming the file to landingLights2 instead of landingLights? Is the old file still used? Asking out of curiosity.

**Review by MadByteDE — APPROVED** [review]
Some comments on minor things. Looks fine to me.

**Pull request merged**